imports/region/Region.js

```diff
@@ -399,31 +362,13 @@ class Region extends Component {
             height={477}
             ref={(node) => {
               this.stage = node;
-              // if (this.stage) {
-              //   const canvas = this.stage.node.toCanvas();
-              //   canvas.width = 1274;
-              //   canvas.height = 954;
-              //   canvas.style.width = '482px';
-              //   canvas.style.height = '477px';
-              //   canvas.getContext('2d').scale(2, 2);
-              // }
             }}
           >
             <Layer
               id="layer"
               ref={(node) => {
-                if (node) this.layer = node;
-                // if (this.layer) {
-                // this.layer.getCanvas().context._context.imageSmoothingQuality = 'high';
-                // console.log(this.layer.getContext().scale(1, 1));
-                // this.layer.getCanvas().setWidth(482);
-                // this.layer.getCanvas().setHeight(477);
-                // this.layer.getCanvas().getContext('2d').scale(2, 2);
-                // }
-                // console.log(this.layer.getCanvas());
-                // canvas.setWidth(1000);
-                // canvas.setHeight(500);
-                // canvas.setSize(482, 477);
+                if (node)
```

Comment:
I prefer the following style for 1 line condition, we ever discussed the similar thing about c++ in slack #code channel, and we including @confluence prefer this for c++. But we never discuss much about JavaScript coding style. 

how do you think?
```
if (node) {

}
```
 @julieyao05, do you have opinions?

Reply:
I prefer either the one-line format (as in the original code) or your suggested format.

Reply:
Agreed, I will add the braces, that is consistent with our ESLint file